src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -116,7 +116,7 @@
             shortName = ExomeStandardArgumentDefinitions.ALLELIC_PON_FILE_SHORT_NAME,
             optional = true
     )
-    protected File allelicPONFile;
+    protected File allelicPoNFile;
```

Comment:
Hmm, on second thought, there are violations everywhere, so that changing this to follow the convention might actually make it stick out more...  I think I will fix `allelicPON` -> `allelicPoN` as you suggest, though.